src/SFA.DAS.Commitments.Api.IntegrationTests/DatabaseSetup/Generators/Generator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.Commitments.Api.IntegrationTests.Helpers;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.DatabaseSetup.Generators
+{
+    // prefer composition?
+    public class Generator
+    {
+        protected readonly Random Random = new Random();
+
+        public long[] RandomIdGroups(long firstId, int countOfIds, int maxIdsPerGroup, int maxIdsRequired)
+        {
+            var newApprenticeshipIdsShuffled = Enumerable
```

Comment:
'Enumerable.Range' is used in a number of places in the test project. Could the uses of them be generalised to avoid repitition?

Reply:
This "could" throw if firstId > int.MaxValue

Reply:
Added an Assert check to check if the id supplied is greater than the method can handle.

Reply:
Agreed the first expression of these 2 methods could be extracted out into a method, but on the 'todo' list is to change the current consumer of the first method over to the second method, which would make the extraction obsolete.